pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -34,7 +34,7 @@ self: super: {
   jailbreak-cabal = (disableSharedExecutables super.jailbreak-cabal).override { Cabal = self.Cabal_1_20_0_4; };
 
   # enable using a local hoogle with extra packagages in the database
-  # nix-shell -p "haskellPackages.hoogleLocal (with haskellPackages; [ mtl lens ])"
+  # nix-shell -p "haskellPackages.hoogleLocal { packages = with haskellPackages; [ mtl lens ]; }"
```

Comment:
I couldn’t remember, fortunately my commit from Dec 2016 was pretty descriptive:

```
`packages` is not a function `HaskellPackages -> [Package]`, but rather an
arbitrary set of packages.
That is good, because that way it can be used e.g. to merge package databases
from other versions of the packageset.

It’s done as function with default argument, because otherwise the user has to
use `overrideCabal` to override `packages` in `hoogle.nix`.
```